package.json

```
@@ -16,17 +16,23 @@
     "type": "git",
     "url": "https://github.com/metarhia/metastreams"
   },
-  "main": "./metastreams.js",
+  "main": "metastreams.js",
   "readmeFilename": "README.md",
   "scripts": {
     "test": "npm run lint && metatests ./test",
-    "lint": "eslint ."
+    "lint": "eslint . && prettier -c \"**/*.js\" \"**/*.json\" \"**/*.md\" \".*rc\" \"**/*.yml\"",
+    "fmt": "prettier --write \"**/*.js\" \"**/*.json\" \"**/*.md\" \".*rc\" \"**/*.yml\""
   },
   "engines": {
     "node": ">=8.6.0"
   },
   "devDependencies": {
-    "eslint": "^4.19.1",
-    "metatests": "^0.2.2"
+    "eslint": "^5.12.1",
+    "eslint-config-metarhia": "^6.1.0",
+    "eslint-config-prettier": "^3.6.0",
+    "eslint-plugin-import": "^2.15.0",
+    "eslint-plugin-prettier": "^3.0.1",
+    "metatests": "^0.4.0",
+    "prettier": "^1.16.1"
```

After the changes introduced in v1.16.0 of Prettier, I'd prefer to not use `^` here:
```suggestion
    "prettier": "1.16.x"
```